resources/views/campaigns/modules/_custom.blade.php

```diff
@@ -20,7 +20,7 @@ class="btn2 btn-primary btn-sm"
     <div class="grid grid-cols-1 md:grid-cols-2 xl:grid-cols-3 gap-2 md:gap-4">
         @foreach ($customEntityTypes as $entityType)
             <div class="cell col-span-1 flex">
-                <x-campaigns.module-box :campaign="$campaign" :entityType="$entityType"></x-campaigns.module-box>
+                <x-campaigns.module-box :campaign="$campaign" :entityType="$entityType" :thumbnails="$thumbnails"></x-campaigns.module-box>
```

Comment:
Doing 20-30 foreach for each module is an expensive use of our servers. Only pass the module's thumbnail or null to the component